pre-pages/pgql-2.0-spec.md

```diff
@@ -1513,20 +1664,34 @@ ColonOrIsKeyword ::=    ':'
 
 Colons and `IS` keywords can be used interchangeably.
 
+Note: With GRAPH_TABLE operator only IS is allowed.
```

Comment:
maybe add quotes so that it becomes in-lines code:

Note: With GRAPH_TABLE operator only IS is allowed.
==>
Note: With `GRAPH_TABLE` operator only `IS` is allowed.